aws/resource_aws_api_gateway_resource.go

```diff
@@ -18,6 +19,20 @@ func resourceAwsApiGatewayResource() *schema.Resource {
 		Read:   resourceAwsApiGatewayResourceRead,
 		Update: resourceAwsApiGatewayResourceUpdate,
 		Delete: resourceAwsApiGatewayResourceDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), "/")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected REST-API-ID/RESOURCE-ID", d.Id())
+				}
+				restApiID := idParts[0]
+				resourceID := idParts[1]
+				d.Set("request_validator_id", resourceID)
+				d.Set("rest_api_id", restApiID)
+				d.SetId(resourceID)
```

Comment:
(as in the other PR's) - would it be easier to set the ID and call Read?

Reply:
Noting for future posterity: in these PRs we're just doing the minimum to support resource import (with a consistent format of `REST-API-ID/THING`) -- we can eventually switch to read/update/delete via the resource ID, but this implementation provides the bare minimum needed for import without the unrelated code changes right now.